### ISSUE TITLE
Swagger warnings for optional parameters even when using `property`

### ISSUE THREAD

**Issue by davidwessman** [Apipie/apipie-rails#888]
Hello!

When I specify my return values like this:
```ruby
          api(
            :POST,
            "/v2/namespace/credentials",
            "Generate new credentials"
          )
          returns(code: 200) do
            property(:username, String)
            property(:password, String)
          end
          def create
          end
```

and generate swagger, I still get warnings:
```ruby
The parameter :username is optional but default value is not specified (use :default_value => ...)
The parameter :password is optional but default value is not specified (use :default_value => ...)
```

even though `property` are required by default. I also tried adding `required: true` but still get the errors.

It does generate the correct swagger file as far as I understand:
```json
"responses": {
          "200": {
            "description": "OK",
            "schema": {
              "type": "object",
              "properties": {
                "username": {
                  "type": "string"
                },
                "password": {
                  "type": "string"
                }
              },
              "additionalProperties": false,
              "required": [
                "username",
                "password"
              ]
            }
          }
        },
```

**Comment by davidwessman** [comment]
The `@in_schema` seems to be true and then it wont consider `@param_description.required` which would have returned true:
https://github.com/Apipie/apipie-rails/blob/61ae61d1247543dcb17ebb58267dc5ca0e987c7f/lib/apipie/generator/swagger/param_description/builder.rb#L82

**Comment by davidwessman** [comment]
Fixed in 1.2.2, thank you @shev-vadim-net 🎉
